### PR TITLE
updating the comments on the contact api file and package.json file for formatting prettier

### PR DESCRIPTION
**Pull request by lucpizz**
## Proposed changes

Created post/create API's for contact webpage.

## PR Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/davidsaulrodriguez/shop-portal/blob/master/CONTRIBUTING.md) doc
- [x] I have read the [BRANCHING MODEL](https://github.com/davidsaulrodriguez/shop-portal/blob/master/BRANCHINGMODEL.md) doc
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## PR Type

What types of changes does your code introduce to **Shop Portal**?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code Style Update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (Please describe):

## Current behavior

Issue Number: N/A

## New behavior

Fixes N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by davidsaulrodriguez — DISMISSED** [review]
Comment updates approved!

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "An open source eCommerce platform powered by the MERN sttack. It is designed to allow anyone to host their own online shop with ease.",
   "main": "server.js",
   "scripts": {
-    "format": "prettier {,*/**/}*.{js,jsx,ts,tsx,json,html} --write",
+    "format": "prettier --write \"{,*/**/,*}*.{js,jsx,ts,tsx,json,html}\"",
```
Comment (davidsaulrodriguez) [inline package.json:7]: Thanks for working through this with me last night to make sure this works not only on Linux / WSL but MacOS also!

**Review by davidsaulrodriguez — APPROVED** [review]
Definitely approved!

**Review by Delph-Sunny — APPROVED** [review]
All good

**Pull request merged**